src/Components/Common/Hoc/SubscriberProvider/index.js

```diff
@@ -176,6 +173,70 @@ class SubscriberProvider extends Component {
     }
   }
 
+  updateSubscriberData = subscriberData => {
+    if (!subscriberData) {
+      return
+    }
+    this.setStateAsync({
+      subscriberData: {
+        ...this.state.subscriberData,
+        ...subscriberData,
+      },
+    })
+  }
+
+  unsubscribeUser = async () => {
+    const { subscriberData } = this.state
+    logger.log('Unsubscribing user with id ', subscriberData)
+    await axios.delete(`/subscribers/${subscriberData.id}`)
+    this.updateSubscriberData({
+      ...defaultState.subscriberData,
+    })
+  }
+
+  subscriberUser = async subscriptionData => {
+    if (!subscriptionData) {
+    }
```

Comment:
Req: mmm missing return ?

Reply:
Yes, thanks, good noticed. I'd changed it and added logging